src/pages/LandingPage.py

```diff
@@ -1,5 +1,9 @@
+import time
+
 from src.pages.common.BaseWrapper import BaseWrapper
 from src.pages.webElements.ButtonElement import ButtonElement
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
```

Comment:
are you sure that we need all these imports here?
we use interaction logic in ButtonElement and InputElement

Reply:
I replaced scroll_down logic to the BaseWrapper.